test/api.jl

```diff
@@ -319,3 +319,41 @@ end
     @test p1.data[1].marker_line_width isa Number
 
 end
+
+@testet "subplots" begin
+
+    using PlotlyJS
+
+    labels = ["1st", "2nd", "3rd", "4th", "5th"]
+
+    # Define color sets of paintings
+    night_colors = ["rgb(56, 75, 126)", "rgb(18, 36, 37)", "rgb(34, 53, 101)",
+                    "rgb(36, 55, 57)", "rgb(6, 4, 4)"]
+    sunflowers_colors = ["rgb(177, 127, 38)", "rgb(205, 152, 36)", "rgb(99, 79, 37)",
+                        "rgb(129, 180, 179)", "rgb(124, 103, 37)"]
+    irises_colors = ["rgb(33, 75, 99)", "rgb(79, 129, 102)", "rgb(151, 179, 100)",
+                    "rgb(175, 49, 35)", "rgb(36, 73, 147)"]
+    cafe_colors =  ["rgb(146, 123, 21)", "rgb(177, 180, 34)", "rgb(206, 206, 40)",
+                    "rgb(175, 51, 21)", "rgb(35, 36, 21)"]
+
+    # Create subplots, using "domain" type for pie charts
+    fig = make_subplots(rows=2, cols=2, specs=fill(Spec(kind="domain"), 2, 2))
+
+    # Define pie charts
+    add_trace!(fig, pie(labels=labels, values=[38, 27, 18, 10, 7], name="Starry Night",
+                        marker_colors=night_colors), row=1, col=1)
+    add_trace!(fig, pie(labels=labels, values=[28, 26, 21, 15, 10], name="Sunflowers",
+                        marker_colors=sunflowers_colors), row=1, col=2)
+    add_trace!(fig, pie(labels=labels, values=[38, 19, 16, 14, 13], name="Irises",
+                        marker_colors=irises_colors), row=2, col=1)
+    add_trace!(fig, pie(labels=labels, values=[31, 24, 19, 18, 8], name="The Night Café",
+                        marker_colors=cafe_colors), row=2, col=2)
+
+    # Tune layout and hover info
+    restyle!(fig, hoverinfo="label+percent+name", textinfo="none")
+    relayout!(fig, title_text="Van Gogh: 5 Most Prominent Colors Shown Proportionally",
+            showlegend=false)
+
+    @test fig.data[1].domain isa Dict
```

Comment:
Hey @tlyon3 I don't think this is sufficient because by default if any property is missing in a trace, we set the default value to Dict when trying to access it.

I would check that some of the expected dict keys exist instead